app/main.ipynb

```diff
@@ -25,18 +25,16 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
-   "outputs": [],
    "source": [
-    "# write your code here"
+    "import sys\n",
+    "import numpy as np\n",
+    "from numpy import dtype"
```

Comment:
The import statement `from numpy import dtype` is unnecessary since `dtype` is not used explicitly in the code. You can remove this line to clean up the imports.